Options.py

```diff
@@ -1115,6 +1115,97 @@ def verify(self, world: typing.Type[World], player_name: str, plando_options: "P
             link.setdefault("link_replacement", None)
 
 
+class PlandoItem(typing.NamedTuple):
+    items: typing.Union[typing.List[str], typing.Dict[str, typing.Any]]
+    locations: typing.Union[typing.List[str]]
+    world: typing.Union[int, str, bool, None, typing.Iterable[str], typing.Set[int]] = False
+    from_pool: bool = True
+    force: typing.Union[bool, typing.Literal["silent"]] = "silent"
+    count: typing.Union[int, bool, typing.Dict[str, int]] = False
+    percentage: int = 100
+
+
+class PlandoItems(Option[typing.List[PlandoItem]]):
+    """Generic items plando."""
+    default = ()
+    supports_weighting = False
+    display_name = "Plando Items"
+
+    def __init__(self, value: typing.Iterable[PlandoItem]) -> None:
+        self.value = list(deepcopy(value))
+        super().__init__()
+
+    @classmethod
+    def from_any(cls, data: typing.Any) -> Option[typing.List[PlandoItem]]:
+        if not isinstance(data, typing.Iterable):
+            raise Exception(f"Cannot create plando connections from non-Iterable type, got {type(data)}")
```

Comment:
Gonna assume this was supposed to be "items"
```suggestion
            raise Exception(f"Cannot create plando items from non-Iterable type, got {type(data)}")
```